.editorconfig

```
@@ -0,0 +1,248 @@
+root = true
+###############################
+# Core EditorConfig Options   #
+###############################
+# All files
+# [*]                            # Do not apply to all files not to break something
+# Either crlf | lf, default is system-dependent (when not specified at all)
+# end_of_line=crlf
+# Remove whitespace at the end of any line
+
+# Code files
+[*.{cs,csx,fs,fsx,vb,vbx}]
+trim_trailing_whitespace = true
+insert_final_newline     = true
+indent_style             = space # default=space
+indent_size              = 4     # default=4
+charset                  = utf-8
+
+# Project files
+[*.{csproj,fsproj,vbproj,shproj,projitems,props,plist}]
+trim_trailing_whitespace = true
+insert_final_newline     = true
+indent_style             = tab   # Visual Studio changes to tabs for old projects
```

I wouldn’t do that. In VS, if you change something through Project Properties, it’ll insert new lines with 2 spaces indent, not tabs. When you edit it by hand, VS tries to turn it into tabs instead. So this rule should be inverted, so that VS will format correctly like the auto tooling when editing by hand:
```suggestion
indent_style             = space
```